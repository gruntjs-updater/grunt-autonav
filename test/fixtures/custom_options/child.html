<div id="nav">
	<div><a href="index.html">Home</a></div>
	<div><a href="page.html">Page</a>
		<div><a href="child.html">Child</a></div>
	</div>
	<div><a href="blog.html">Blog</a></div>
	<div><a href="contact.html">Contact</a></div>
</div>
