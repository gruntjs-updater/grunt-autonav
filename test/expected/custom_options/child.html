<div id="nav">
	<div><a href="index.html">Home</a></div>
	<div class="current-menu-ancestor"><a href="page.html">Page</a>
		<div class="current-menu-item"><a href="child.html">Child</a></div>
	</div>
	<div><a href="blog.html">Blog</a></div>
	<div><a href="contact.html">Contact</a></div>
</div>
